doc/topics/orchestrate/orchestrate_runner.rst

```diff
@@ -38,22 +36,34 @@ application of states on different minions that must not happen simultaneously,
 or for halting the state run on all minions if a minion fails one of its
 states.
 
-If you want to setup a load balancer in front of a cluster of web servers, for
-example, you can ensure the load balancer is setup before the web servers or
-stop the state run altogether if one of the minions does not set up correctly.
+If you want to set up a load balancer in front of a cluster of web servers, for
+example, you can ensure the load balancer is set up before the web servers or
+stop the state run altogether if a minion fails to apply any states.
 
-The ``state.sls``, ``state.highstate``, et al functions allow you to statefully
+The ``state.sls``, ``state.highstate``, et al. functions allow you to statefully
 manage each minion and the ``state.orchestrate`` runner allows you to
 statefully manage your entire infrastructure.
 
+Writing SLS Files
+~~~~~~~~~~~~~~~~~
+
+Orchestrate SLS files are stored in the same location as State SLS files. This
+means that both ``file_roots`` and ``gitfs_remotes`` impact what SLS files are
+available to the reactor and orchestrator.
+
+It is recommended to keep reactor and orchestrator SLS files in their own uniquely
+named subdirectories such as ``_orch/``, ``orch/``, ``_orchestrate/``, ``react/``,
+``_reactor/``, etc. Keeping a unique name helps prevent confusion when trying to
+read through this a few years down the road.
```

Comment:
Agreed, it's a bit of a run-on. :+1:

Any thoughts on replacing the whole last line with this?
`This will avoid duplicate naming and will help prevent confusion.`